test/test_sparse.py

```diff
@@ -2386,7 +2386,7 @@ def test_sparse_to_numpy(self):
         t = torch.sparse_coo_tensor(torch.tensor(([0, 0], [2, 0])), torch.tensor([1, 4]))
         self.assertRaises(TypeError, lambda: t.numpy())
 
-    @cpu_only
+    @skipIfRocm
```

Comment:
In the future it'd be interesting to consider adding tests that validated the cpu and CUDA produced consistent results, too.

Reply:
I think this test covers both cases as method `[_gen_sparse](https://github.com/pytorch/pytorch/blob/1ccdc91175e9900d9103081bb9d0f03c178ed694/test/test_sparse.py#L2599)`  uses `self.device` variable to generate sparse tensor with cpu and cuda configurations.